test/e2e/pkg/client/pod.go

```diff
@@ -99,3 +98,18 @@ func (w *K8sClient) isPodRunning(podName, namespace string) wait.ConditionFunc {
 		return false, nil
 	}
 }
+// return the pod name from the dev user namespace (config.DevNameSpace)
+func (w *K8sClient) GetPodNameFromUserNameSpaceByLabel(selector string)  string {
+	podList, err := w.ListPods(config.DevNameSpace, selector)
+
+	if err != nil {
+		log.Fatal("Cannot obtain pods from developer namespace " + err.Error())
```

Comment:
Should we include the namespace from config.DevNamespace here? Instead of saying developer namespace

Reply:
Done